### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by regisss**
I spotted a few typos while reading the "Optimizing your LLM in production" blog post. Very nice article!

### CODE REVIEW

File: optimize-llm.md [hunk 1]
```diff
@@ -574,14 +574,14 @@ As shown in the [ALiBi](https://arxiv.org/abs/2108.12409) paper, this simple rel
...
 For ALiBi, one simply increases the values of the lower triangular position matrix to match the length of the input sequence.
-For *RoPE*, keeping the same \\( \theta \\) that was used during training leads to poor results when passing text inputs much longer than those seen during training, *c.f* [Press et al.](https://arxiv.org/abs/2108.12409). However, the community has found a couple of effective tricks that adapt \\( \theta \\) . thereby allowing *RoPE* position embeddings to work well for extrapolated text input sequences (see [here](https://github.com/huggingface/transformers/pull/24653)).
+For *RoPE*, keeping the same \\( \theta \\) that was used during training leads to poor results when passing text inputs much longer than those seen during training, *c.f* [Press et al.](https://arxiv.org/abs/2108.12409). However, the community has found a couple of effective tricks that adapt \\( \theta \\). Thereby allowing *RoPE* position embeddings to work well for extrapolated text input sequences (see [here](https://github.com/huggingface/transformers/pull/24653)).
```
Comment (pcuenca) [inline optimize-llm.md:577]:
```suggestion
For *RoPE*, keeping the same \\( \theta \\) that was used during training leads to poor results when passing text inputs much longer than those seen during training, *c.f* [Press et al.](https://arxiv.org/abs/2108.12409). However, the community has found a couple of effective tricks that adapt \\( \theta \\), thereby allowing *RoPE* position embeddings to work well for extrapolated text input sequences (see [here](https://github.com/huggingface/transformers/pull/24653)).
```

**Review by pcuenca — APPROVED** [review]
Awesome, thanks! There's an additional broken image in line 566 (alibi.png), would it be easy for you to add it this PR? (otherwise I'll do later :)

**Comment by regisss** [comment]
> Awesome, thanks! There's an additional broken image in line 566 (alibi.png), would it be easy for you to add it to this PR? (otherwise I'll do later :)

Thanks for the review @pcuenca ! I'm going to fix the link to this image now :slightly_smiling_face:

**Comment by regisss** [comment]
@pcuenca Done!
I also corrected the link to the thumbnail, even though it seems to work with the wrong path for whatever reason :thinking:

**Review by pcuenca — APPROVED** [review]
Thanks a lot! 🙌

**Comment by pcuenca** [comment]
Merging, cc @patrickvonplaten

**Pull request merged**